### ISSUE TITLE
Highcharts elements announce as "clickable" with NVDA

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#24151]
#### Expected behaviour
Non-clickable elements should not announce as "clickable".

#### Actual behaviour
Many elements announce as "clickable".

Most likely this is due to Highcharts attaching event handlers to the DOM objects, and NVDA picks these up as JS making the objects clickable, even if this is not the actual effect of the DOM handlers. See https://github.com/nvaccess/nvda/issues/5830. 

It may be limited what we can do about this without harming interactivity, but we should look into ways to not add these events unless necessary for interactivity. As a minimum, we should investigate if there is a workaround. 

https://www.highcharts.com/demo/highcharts/advanced-accessible tries to work around the issue, but apparently is not successful anymore. See https://www.highcharts.com/forum/viewtopic.php?t=46800 for context.

**Comment by magdalena-gut** [comment]
After investigating several approaches, none of which could be integrated into the core without regressions, we settled on a standalone plugin as a workaround. It hooks into afterA11yUpdate to remove unnecessary event handlers from the chart container. Closing this issue with the workaround below:

```
(function (H) {
    H.addEvent(H.Chart, 'afterA11yUpdate', function () {
        const chart = this;

        if (!chart.pointer) {
            return;
        }

        const container = chart.container,
            chartOptions = chart.options.chart,
            zoomType = chart.zooming.type || '',
            hasZoom = /x|y/.test(zoomType),
            panning = chartOptions.panning,
            hasPanning = H.isObject(panning) ? !!panning.enabled : !!panning,
            hasDraggableNodes = chart.series.some(
                (s) => !!s.hasDraggableNodes
            );

        if (!hasZoom && !hasPanning && !chartOptions.events?.selection && !hasDraggableNodes) {
            container.onmousedown = null;
            H.removeEvent(container, 'mousedown');
        }

        if (!chart.navigationBindings && !chartOptions.events?.click && !chart.runTrackerClick) {
            container.onclick = null;
        }
    });
}(Highcharts));
```

Demo example:
https://jsfiddle.net/BlackLabel/3u7hrwcn/
